packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx

```diff
@@ -112,14 +105,27 @@ const SideBar: React.FunctionComponent<SideBarProps> = ({
         >
           {framework
             ? `${framework} does not support Slice Preview.`
-            : "Slice Preview is not supported by your framework."}{" "}
-          You can{" "}
-          <a target={"_blank"} href={linkToStorybookDocs}>
-            install Storybook
-          </a>{" "}
-          instead.
+            : "Slice Preview is not supported by your framework."}
+          &nbsp;
+          {!storybookUrl ? (
```

Comment:
OPT: you can use `condition && thiing` instead of  `condition ? thing : null` :)

Reply:
I can, it's just that multiple times i've had `thing.length && Element` displaying a `0` so i prefer to set it explicitely.